src/Microsoft.ML.StandardLearners/Standard/LinearClassificationStatic.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.Internal.Calibration;
+using Microsoft.ML.Runtime.Internal.Internallearn;
+using Microsoft.ML.Runtime.Learners;
+using Microsoft.ML.StaticPipe.Runtime;
+
+namespace Microsoft.ML.StaticPipe
+{
+    using Arguments = StochasticGradientDescentClassificationTrainer.Arguments;
+
+    /// <summary>
+    /// Binary Classification trainer estimators.
+    /// </summary>
+    public static partial class BinaryClassificationTrainers
+    {
+        /// <summary>
+        ///  Predict a target using a linear binary classification model trained with the <see cref="Microsoft.ML.Runtime.Learners.StochasticGradientDescentClassificationTrainer"/> trainer.
+        /// </summary>
+        /// <param name="ctx">The binary classificaiton context trainer object.</param>
+        /// <param name="label">The name of the label column.</param>
+        /// <param name="features">The name of the feature column.</param>
+        /// <param name="weights">The name for the example weight column.</param>
+        /// <param name="maxIterations">The maximum number of iterations; set to 1 to simulate online learning.</param>
+        /// <param name="initLearningRate">The initial Learning Rate used by SGD.</param>
```

Comment:
>Learning R [](start = 55, length = 10)

nit: no need for title case

Reply:
Thank you!

---
In reply to: [222897917](https://github.com/dotnet/machinelearning/pull/1134#discussion_r222897917) [](ancestors = 222897917)